tools/st2_deploy.sh

```diff
@@ -97,6 +99,22 @@ else
     MISTRAL_STABLE_BRANCH="st2-0.5.1"
 fi
 
+update_hosts_file()
+{
+  if [ -z "`cat /etc/hosts | grep -e '127.0.0.1\slocalhost'`" ]; then
```

Comment:
Anyway, I found a fix. The problem is in the st2client default base URL.  I'm going to set it to http://0.0.0.0 (currently it's http://localhost).  That fixed the problem.